cli/src/commands/branch/set.rs

```diff
@@ -84,5 +82,18 @@ pub fn cmd_branch_set(
             target_commit.id().hex()
         ),
     )?;
+
+    if !new_branch_names.is_empty() {
+        writeln!(
+            ui.status(),
+            "Created branches: {}",
+            new_branch_names.join(", "),
+        )?;
+        // TODO: delete this hint in jj 0.25+
+        writeln!(
```

Comment:
(Non-blocking) I'm still wondering whether there should be an option to disable the hint, but I guess it's quite possible we don't. We can add it if people ask for it, I suppose; it doesn't have to be in this PR.